catalog/solr/catalog-solr-core/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -218,7 +220,7 @@ public SourceResponse query(QueryRequest request) throws UnsupportedQueryExcepti
       QueryResponse solrResponse;
       start = System.nanoTime();
       LOGGER.trace("Begin executing solr query for {}", traceSource);
-      if (shouldDoRealTimeGet(request)) {
+      if (!isFacetedQuery && shouldDoRealTimeGet(request)) {
```

Comment:
✏️ Might be worth a comment in the code as to why faceted queries don't do real time get requests.  You could also do this check inside the `shouldDoRealTimeGet` method so that all of the logic is captured there.  Not required since it already figures out here if it is a faceted query but perhaps something to think about from an organization perspective.